da-indexer/da-indexer-server/config/celestia.toml

```diff
@@ -35,6 +35,7 @@ polling_interval = 12
 
 [indexer.da]
 type = "Celestia"
+save_batch_size = 1000
 
 [indexer.da.rpc]
```

Comment:
Should be:
```suggestion
[indexer.da.celestia]
save_batch_size = 1000

[indexer.da.celestia.rpc]
```

Reply:
Done in https://github.com/blockscout/blockscout-rs/pull/1307/commits/440622f7b650cda516fd1f766e8662240bfb6640